rate-providers/LoopRateProvider.md

```diff
@@ -0,0 +1,57 @@
+# Rate Provider: LoopRateProvider
+
+## Details
+- Reviewed by: @brunoguerios
+- Checked by: @\<GitHub handle of secondary reviewer\>
+- Deployed at:
+    - [ethereum:0x1f037c849CF2448d67A120543EA4ec3CE5A95FcA](https://etherscan.io/address/0x1f037c849CF2448d67A120543EA4ec3CE5A95FcA)
+    - [\<audit title\>](\<link to audit\>)
+
+TODO: I found some audits on their site, but related to a different contract - seems like it would make more sense for them to fill out these links instead of us trying to find them?
+
+## Context
+\<Write a brief description of the intended functionality here.\>
+
+## Review Checklist: Bare Minimum Compatibility
+Each of the items below represents an absolute requirement for the Rate Provider. If any of these is unchecked, the Rate Provider is unfit to use.
+
+- [x] Implements the [`IRateProvider`](https://github.com/balancer/balancer-v2-monorepo/blob/bc3b3fee6e13e01d2efe610ed8118fdb74dfc1f2/pkg/interfaces/contracts/pool-utils/IRateProvider.sol) interface.
+- [x] `getRate` returns an 18-decimal fixed point number (i.e., 1 == 1e18) regardless of underlying token decimals.
```

Comment:
getRate does return 18 decimals because the erc4626 provided is 18 decimals - if erc4626 was 6 decimals, getRate would return 6 decimals - is that a problem?

Reply:
`getRate` must always return 18 decimal fixed point number. The ERC4626 Rate providers deployed from the factory, account for the potential of different decimals. See here: https://github.com/balancer/metastable-rate-providers/blob/master/contracts/ERC4626RateProvider.sol#L34.

Reply:
ok, so this is incorrect - should we simply mark this as non-compatible? or do we sync with loop team and ask them to provide a new rate provider?